yoyodyne/models/modules/transformer.py

```diff
@@ -336,21 +336,21 @@ def forward(
                 encoder.
             features_memory (torch.Tensor): the mask for the features.
             target_mask (Optional[torch.Tensor], optional): the mask for the
-                target sequence. Defaults to None.
+                target sequence.
             memory_mask (Optional[torch.Tensor], optional): the mask for the
-                memory sequence. Defaults to None.
+                memory sequence.
             features_memory_mask (torch.Tensor, optional): the mask
```

Comment:
Should this be `Optional[torch.Tensor]` ?

Reply:
Fixed